src/base/PlatypusApp.C

```diff
@@ -47,6 +47,7 @@ associateSyntaxInner(Syntax & syntax, ActionFactory & /*action_factory*/)
 
   // add FESpaces
   registerMooseObjectTask("add_mfem_fespaces", MFEMFESpace, false);
+  appendMooseObjectTask("add_mfem_fespaces", MFEMFECollection);
```

Comment:
A user may try to create an `MFEMFECollection` directly in the `FESpaces` input block. This should be unlikely to cause issues, provided `MFEMFESpaces` do not depend on any `MFEMFECollection` built in this manner.